docs/guides/user-guides/user-quickstart-guide.md

```diff
@@ -0,0 +1,67 @@
+User Quickstart Guide
+====================
+
+Read the User Quickstart guide to gain a brief understanding of key concepts and how to use the tools. For more in-depth
+information, consult [OpenMS API Reference](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/index.html).
+
+## What is OpenMS
+
+[OpenMS](https://www.openms.de/) is a free, open-source C++ library with Python bindings. It is commonly used for liquid
+chromatography-mass spectrometry ({term}`LC-MS`) data management and analyses. OpenMS provides an infrastructure for the rapid
+development of mass spectrometry related software as well as a rich toolset built on top of it. OpenMS is available
+under the [three clause BSD licence](https://github.com/OpenMS/OpenMS/blob/develop/LICENSE) and runs under Windows, macOS, and Linux operating systems.
+
+## Background
+
+Before using OpenMS, become familiar with the following terms:
+
+| Tool and Utilities | Description |
+|--------------------|-------------|
+|**TOPPView**        |A design tool that is used to view and explore {term}`LC-MS` data, alignments, groups, peptide identifications, and more.|
+|**TOPPAS**          |A graphical workflow design tool that is used to create pipelines from all {term}`TOPP tools` (and {term}`UTILS`).|
```

Comment:
Not sure if TOPPAS should be that prominent in the quickstart guide if the general recommendation is to use KNIME as a graphical workflow manager.

Reply:
Agree, we probably should "hide" it a bit